frontend/talentsearch/src/js/components/experienceForm/ExperienceSkills.tsx

```diff
@@ -51,6 +57,18 @@ const ExperienceSkills: React.FC<ExperienceSkillsProps> = ({ skills }) => {
     replace(massagedSkills);
   };
 
+  const handleAddSkill = (id: string) => {
+    const skillToAdd = skills.find((skill) => skill.id === id);
+
+    if (skillToAdd) {
+      append({
+        skillId: skillToAdd.id,
+        name: skillToAdd.name,
+        details: "",
+      });
```

Comment:
I'm going to leave the descision up to @Jerryescandon , but I think we may prefer to add `{shouldFocus: false}` to the `append`.

@Jerryescandon What do you think? By default, when you add a skill it jumps down the to details section to start writing, do we want to prevent that so users don't need to scroll back up to add more skills?